flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/batch/sql/TableSourceITCase.scala

```diff
@@ -247,4 +248,24 @@ class TableSourceITCase extends BatchTestBase {
         row(3, "7.10", "123", "123"))
     )
   }
+
+  @Test
+  def testMultiPaths(): Unit = {
+    val tmpFile1 = File.createTempFile("flink-table-sink-test", ".tmp")
```

Comment:
Write some records to the two files and verify it‘s works.

Reply:
done, thanks for your review.